lib/fpm/command.rb

```diff
@@ -35,10 +35,14 @@ def help(*args)
       "  You can find support on irc (#fpm on freenode irc) or via email with",
       "  fpm-users@googlegroups.com",
       "",
-
-      # Lastly, include the default help output via Clamp.
-      super
-    ].join("\n")
+      "Loaded package types:",
```

Comment:
I'm tempted to change this to "Available package types" but will wait for user confusion to change it.